packages/sample/src/main.ts

```diff
@@ -62,18 +75,17 @@ program
     .command("list")
     .description("List all connected expansion hubs")
     .action(async () => {
-        await list();
+        let hubs = await openConnectedExpansionHubs();
+        await list(hubs);
     });
 
 program
     .command("led")
     .description("Run LED steps")
     .action(async () => {
-        let hubs = await openConnectedExpansionHubs();
-        let hub = hubs[0];
-
+        let [hub, close] = await getExpansionHubOrThrow();
         runOnSigint(() => {
-            hub.close();
+            close();
         });
 
         await led(hub);
```

Comment:
The led method is different because it continues execution after setting the pattern. If we called close when it exits, the pattern would immediately stop. This is similar to runServo.